pkg/service/pipeline.go

```diff
@@ -1985,7 +1985,7 @@ func (s *service) ListComponentRuns(ctx context.Context, req *pipelinepb.ListCom
 
 	return &pipelinepb.ListComponentRunsResponse{
 		ComponentRuns: pbComponentRuns,
-		TotalSize:     totalCount,
+		TotalSize:     int32(totalCount),
```

Comment:
@donch1989 I didn't overthink the changes in this commit, I just wanted to fix the build after the breaking changes in https://github.com/instill-ai/protobufs/commit/e04c85940bfb1927202afe614cdda9623b5f5b44 and https://github.com/instill-ai/protobufs/commit/9f778f002b7289e0900aea75788e299be014ec7c.